index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Andika&family=Inter:wght@300;400;500;700&family=Poppins:ital,wght@0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,200&family=Roboto:wght@100;300;400;500;700;900&family=Ubuntu:ital,wght@1,300&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<header class="container">
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <a href="#">My Logo</a>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+        <button class="hamburger"><i class="fa fa-bars"></i></button>
+    </div>
```

Comment:
- [ ] Great work aligning your hamburger menu button, well done! Let's make it even better by using the template's color and removing the surrounding border which is not part of the design, try with `border` or `outline` properties to remove it:

Template:
![Captura de pantalla 2022-04-23 a la(s) 12 10 16 a  m](https://user-images.githubusercontent.com/83717285/164878783-15e57e52-ed84-4d41-89a2-8d40eb830ae0.png)


Your design:
![Captura de pantalla 2022-04-23 a la(s) 12 10 24 a  m](https://user-images.githubusercontent.com/83717285/164878787-fa8621e0-12b2-43bf-b51c-3304fd2e64ec.png)